spec/spec_helper.rb

```diff
@@ -2,6 +2,8 @@
 
 require 'heavy_keeper'
 require 'mock_redis'
+require 'redis'
+require 'byebug'
```

Comment:
Hm! Is this line needed? I would have thought it would require automatically.

Reply:
Yes, we need it. It's only required automatically if we use `require 'bundler/setup'`

Reply:
Ohhh. TIL! 🙏